xls/examples/hsv2rgb.x

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Fast HSV to RGB conversion.
+//
+// Range:
+//   h: [0, 0x5ff], s: [0, 0xff], v: [0, 0xff]
+//   r: [0,  0xff], g: [0, 0xff], b: [0, 0xff]
+//
+// Ported from https://www.vagrearg.org/content/hsvrgb (MIT license).
```

Comment:
Given that this was derived from code licensed by MIT -- I think we need a copy of the MIT license here rather than just a web link.